api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/PublishNodesEndpointApiModel.cs

```diff
@@ -14,48 +15,53 @@ namespace Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models {
     [DataContract]
     public class PublishNodesEndpointApiModel {
 
-        /// <summary> The Group the stream belongs to - DataSetWriterGroup. </summary>
+        /// <summary> The Group the stream belongs to - DataSetWriterGroup.</summary>
         [DataMember(Name = "dataSetWriterGroup", Order = 0,
             EmitDefaultValue = false)]
         public string DataSetWriterGroup { get; set; }
 
-        /// <summary> Id Identifier of the DataFlow - DataSetWriterId. </summary>
+        /// <summary> Id Identifier of the DataFlow - DataSetWriterId.</summary>
         [DataMember(Name = "dataSetWriterId", Order = 1,
             EmitDefaultValue = false)]
         public string DataSetWriterId { get; set; }
 
-        /// <summary> The Publishing interval for a dataset writer </summary>
+        /// <summary> The Publishing interval for a dataset writer.</summary>
         [DataMember(Name = "dataSetPublishingInterval", Order = 2,
             EmitDefaultValue = false)]
         public int? DataSetPublishingInterval { get; set; }
 
+        /// <summary> The Publishing interval for a dataset writer as TimeSpan.</summary>
+        [DataMember(Name = "dataSetPublishingIntervalTimespan", Order = 3,
+            EmitDefaultValue = false)]
+        public TimeSpan? DataSetPublishingIntervalTimespan { get; set; }
```

Comment:
Please add tests that would exercise setting this field and getting value back from it. Currently there seem to be no test that uses it.

Reply:
done